dataflow/src/main/java/org/checkerframework/dataflow/analysis/ForwardAnalysisImpl.java

```diff
@@ -344,7 +344,13 @@ protected void initInitialInputs() {
     UnderlyingAST underlyingAST = cfg.getUnderlyingAST();
     List<LocalVariableNode> parameters = getParameters(underlyingAST);
     assert transferFunction != null : "@AssumeAssertion(nullness): invariant";
-    S initialStore = transferFunction.initialStore(underlyingAST, parameters);
+    S initialStore;
+    try {
+      initialStore = transferFunction.initialStore(underlyingAST, parameters);
+    } catch (Exception e) {
+      throw new Error(
```

Comment:
These should be BugInCF, which we do use in the dataflow package. (It's declared in the javacutil package.)